bench-vortex/src/lib.rs

```diff
@@ -167,49 +165,30 @@ pub fn setup_logger(level: LevelFilter) {
     .unwrap();
 }
 
-pub fn compress_taxi_data() -> Array {
+pub fn fetch_taxi_data() -> Array {
     let file = File::open(taxi_data_parquet()).unwrap();
     let builder = ParquetRecordBatchReaderBuilder::try_new(file).unwrap();
-    let _mask = ProjectionMask::roots(builder.parquet_schema(), [6]);
-    let _no_datetime_mask = ProjectionMask::roots(
-        builder.parquet_schema(),
-        [0, 3, 4, 5, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18],
-    );
-    let reader = builder
-        .with_projection(_mask)
-        //.with_projection(no_datetime_mask)
-        .with_batch_size(BATCH_SIZE)
-        // .with_batch_size(5_000_000)
-        // .with_limit(100_000)
-        .build()
-        .unwrap();
+    let reader = builder.with_batch_size(BATCH_SIZE).build().unwrap();
```

Comment:
nit: i believe this is already the DEFAULT_BATCH_SIZE for the reader

Reply:
The Parquet crate [claims otherwise](https://github.com/apache/arrow-rs/blob/master/parquet/src/arrow/arrow_reader/mod.rs#L111-L117):

```
    /// Set the size of [`RecordBatch`] to produce. Defaults to 1024
    /// If the batch_size more than the file row count, use the file row count.
    pub fn with_batch_size(self, batch_size: usize) -> Self {
```

We define [BATCH_SIZE as](https://github.com/spiraldb/vortex/blob/develop/bench-vortex/src/reader.rs#L43):
```
pub const BATCH_SIZE: usize = 65_536;
```

Reply:
You're right, I confused this with our LayoutReaderBuilder